vars/main.yml

```diff
@@ -11,5 +11,8 @@ go_arch: "{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }
 promtail_dependencies:
   - unzip
 
-promtail_system_group: "promtail"
+
 promtail_system_user: "promtail"
+promtail_system_group: "{{ promtail_system_user }}"
+promtail_user_optional_groups:
+  - adm
```

Comment:
What permissions does “adm” group have?

It is necessary to grant promtail minimum possible permissions. So, if it is compromised, the attacker would be able to read log files and that’s all.

An name of group “adm” looks like it gives more permissions than necessary.

Reply:
I was concerned about that too,  but `adm` group just has access to read log files and xconsole    https://wiki.debian.org/SystemGroups#:~:text=adm%3A%20Group%20adm%20is%20used,the%20name%20of%20the%20group.

Reply:
Great! I didn't know about `adm` group!

Have you tested if non-root user is able to grab apache and miarec_speech log files?
We use GPU instances that are based on Nvidia AMI, where umask is very restrictive (722). As a result, the created log files are not visible to other users.

Reply:
I was able to successfully test apache and miarec_speech logs, I did not test with the GPU instances, because my sandbox account had some limits that wouldnt allow me to deploy them,  so i used an older version to test with

Reply:
Old version had a more permissive umask.
You can request an increase of vCPU limits for G instances from Amazon. It takes 1-2 days for them to approve.

Reply:
GPU instance has been tested successfully.

In the ASR, i verified that asr_server.log was read by promtail.

In the web instance I notived the umask is 0002, so promtail has permission to read logs when in adm group, just to verify  I did set the umask to 0722 and verified that apache logs and the housekeeper logs were still read by promtail